src/graph-helpers/join.js

```diff
@@ -0,0 +1,3 @@
+export default function join(...fields) {
+  return fields.join(', ');
+}
```

Comment:
This is for joining queries. Not fields. I realized just now it looks weird right next to the fields helper. It should probably be used there.